### PR TITLE
Bug1654682 5.7

### PR DESCRIPTION
**Pull request by laurynas-biveinis**
http://jenkins.percona.com/job/mysql-5.7-param/697/

### CODE REVIEW

File: mysql-test/suite/rpl/t/rpl_gtid_read_only_flush_logs.test [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+--echo #
+--echo # Bug 84350: Error 1290 executing flush logs in read-only slave
+--echo #
+
+CREATE USER 'reload_user'@'localhost';
+GRANT RELOAD ON *.* TO 'reload_user'@'localhost';
+
+SET @saved_read_only= @@GLOBAL.read_only;
+SET GLOBAL read_only= @saved_read_only;
```
Comment (percona-ysorokin) [inline mysql-test/suite/rpl/t/rpl_gtid_read_only_flush_logs.test]: The same as in 5.6
`SET GLOBAL read_only= ON` ?

**Review by percona-ysorokin — CHANGES_REQUESTED** [review]

**Comment by laurynas-biveinis** [comment]
Fixed http://jenkins.percona.com/job/mysql-5.7-param/700/

**Review by robgolebiowski — APPROVED** [review]
LGTM

**Review by percona-ysorokin — APPROVED** [review]
LGTM

**Pull request merged**